src/components/top-bar.js

```diff
@@ -36,16 +37,22 @@ const TopBar = (props) => {
 
     const classes = useStyles();
 
+    const history = useHistory();
+
     const onParametersClick = () => {
       if (props.onParametersClick) {
           props.onParametersClick();
       }
     };
 
+    const onLogoClick = () => {
+        history.replace("/");
+    }
+
     return (
         <AppBar position="static" color="default" className={classes.appBar}>
             <Toolbar>
-                <PowsyblLogo className={classes.logo}/>
+                <PowsyblLogo className={classes.logo} onClick={onLogoClick}/>
                 <Typography variant="h6" className={classes.title}>
                     <FormattedMessage id="appName"/>
                 </Typography>
```

Comment:
Wouldn't it be possible to create a clickable field that includes both logo and name instead?

Reply:
Done